### PR TITLE
docs: Update part about DCOR

### LINKED ISSUE #182: Update docs on how to access data on a different DCOR server
URLs are as follows:

https://dcor.mpl.mpg.de/api/3/action/dcserv?id=6fd1b41c-7ee4-47e4-93d5-1cc05d4d43d2

### PR DESCRIPTION
**Pull request by B-Hartmann**
Update the docs about DCOR: How to manage API Tokens and how to access data on a different DCOR instance.

Closes #182

### CODE REVIEW

File: docs/sec_av_dcor.rst [hunk 1]
```diff
@@ -38,23 +38,60 @@ field in (*Additional Information*). The DCOR format is documented in
...
     APIHandler.add_api_key("XXXX-YYYY-ZZZZ")
     ds = dclab.new_dataset("fb719fb2-bd9f-817a-7d70-f4002af916f0")
+
+Managing API Tokens
+===================
+
+You can manage your API Tokens on your profile page when
+logged in at https://dcor.mpl.mpg.de.
+
+.. figure:: figures/API_Token_website.PNG
```
Comment (paulmueller) [inline docs/sec_av_dcor.rst:68]: please use lower-case suffixes (.png)

**Comment by paulmueller** [comment]
Thanks for updating the docs. I just had a minor comment. Let me know whether you would like to change anything, else I will merge :pray:

File: docs/sec_av_dcor.rst (hunk 1 above)
Comment (B-Hartmann) [reply docs/sec_av_dcor.rst]: Changed it.

**Comment by B-Hartmann** [comment]
I updated the image with the part about API tokens from the DCOR website to show some more details. This is ready to be merged!

**Review by paulmueller — CHANGES_REQUESTED** [review]
Please also rename the PNG file. Otherwise building the docs will fail on non-Windows system, since POSIX paths are case-sensitive.

**Review by paulmueller — APPROVED** [review]

**Pull request merged**